openrtb_ext/bidders.go

```diff
@@ -368,6 +369,7 @@ const (
 	BidderCadentApertureMX  BidderName = "cadent_aperture_mx"
 	BidderCcx               BidderName = "ccx"
 	BidderCoinzilla         BidderName = "coinzilla"
+	BidderCointraffic       BidderName = "cointraffic"
```

Comment:
Super nitpick: please move this above `BidderCoinzilla` to maintain alphabetical order.